packages/@webex/plugin-meetings/src/member/util.ts

```diff
@@ -226,21 +218,28 @@ MemberUtil.isLiveAnnotationSupported = (participant) => {
 
 /**
  * utility method for audio/video muted status
- * @param {String} status
- * @param {String} accessor
- * @returns {Boolean}
+ * @param {any} participant
+ * @param {String} statusAccessor
+ * @param {String} controlsAccessor
+ * @returns {Boolean | undefined}
  */
-MemberUtil.isMuted = (status: string, accessor: string) => {
-  if (status) {
-    if (status[accessor] === _RECEIVE_ONLY_) {
-      return true;
-    }
-    if (status[accessor] === _SEND_RECEIVE_) {
-      return false;
-    }
+MemberUtil.isMuted = (participant: any, statusAccessor: string, controlsAccessor: string) => {
+  // check remote mute
+  const remoteMute = participant?.controls?.[controlsAccessor]?.muted;
+  if (remoteMute === true) {
+    return true;
   }
 
-  return null;
+  // check local mute
+  const localStatus = participant?.status?.[statusAccessor];
+  if (localStatus === _RECEIVE_ONLY_) {
+    return true;
+  }
+  if (localStatus === _SEND_RECEIVE_) {
+    return false;
+  }
+
+  return remoteMute;
```

Comment:
Looking at this code it seems that `remoteMute` might be undefined and it would be better to always return a boolean here, but I see you have explicit unit tests for this case that check if we return a boolean, so I must be missing something.

Reply:
I was trying to maintain the original behavior.  Not sure why returning undefined was desirable but didn't want to mess with it.  There is code in cantina that assumes muted if undefined.